src/Components/UserSettingsTwoFactorAuthentication/Components/__stories__/BackupSecondFactorMethod.story.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Theme } from "@artsy/palette"
+import { storiesOf } from "@storybook/react"
+import { UserSettingsTwoFactorAuthentication_me } from "__generated__/UserSettingsTwoFactorAuthentication_me.graphql"
+import React from "react"
+import { BackupSecondFactorMethod } from "../BackupSecondFactorMethod"
+
+storiesOf("Settings/2FA/Components", module).add(
+  "BackupSecondFactorMethod/Disabled",
+  () => {
+    const fixture = ({
+      backupSecondFactors: [],
+    } as unknown) as UserSettingsTwoFactorAuthentication_me
```

Comment:
Am not familiar with this, is there a tldr for the `(… as unknown) as …` construct here?

Reply:
This got around the following error but I'm sure there's a better way to structure these fixtures. Probably by using the `@raw_response_type` query decorator and using a `*QueryRawResponse` type.

```
Conversion of type '{ backupSecondFactors: undefined[]; }' to type 'UserSettingsTwoFactorAuthentication_me' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```